plugins/foreman_remote_execution/0.2/index.md

```diff
@@ -31,7 +31,7 @@ A few examples of tasks that this plugin helps you achieve:
 
 - Install/update/remove a package
 - Bootstrap a configuration management agent
-- Trigger a Puppet/Salt/Chef run
+- Trigger a Puppet/Salt/Chef/Ansible run
```

Comment:
Note that the foreman_remote_execution plugin maintains a nightly/ version that's used to cut their next release, you should probably update that (either instead, or as well as, if they want this change).